lib/git_worktree.rb

```diff
@@ -167,6 +167,19 @@ def file_list
     tree.walk(:preorder).collect { |root, entry| "#{root}#{entry[:name]}" }
   end
 
+  # Like "file_list", but doesn't return directories
+  def blob_list &block
```

Comment:
Prefer non-Seattle style.

Reply:
Oops, my bad, will fix.

(For future reference:  I do this personally, but I will fix when rubocop yells.  Didn't run `rubocop` locally before pushing since I wanted to get this in front of faces prior to eating lunch)